src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,16 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "countries")
 public class Country implements Cloneable {
```

Comment:
Implementing Cloneable and providing a clone method is not typical for entity classes. Entities are usually managed by an ORM like Hibernate, which handles their lifecycle, and cloning entities can lead to unexpected behavior, especially when considering lazy-loading and proxies. If you need to create copies of objects, consider using copy constructors or factory methods.